dicebot/handlers/message/hbd_handler.py

```diff
@@ -27,4 +27,4 @@ async def handle(
             await ctx.message.add_reaction(guild_emojis["this_tbh"])
         elif "this_tbh" in all_emojis.keys():
             await ctx.message.add_reaction(all_emojis["this_tbh"])
-        await ctx.channel.send("^")
```

Comment:
Since you're really close to this code, would you mind updating it? It's bothered me a few times.
Change line 14 to be `return ctx.message.content.lower().startswith("hbd")`
Without the `lower()` call, when people say `Hbd` it doesn't work.

Reply:
LOL yeah I can do that